### PR TITLE
Add some loading timing data into the session.devtools_start event

### PR DESCRIPTION
**Pull request by jaril**

### CODE REVIEW

File: src/ui/utils/mixpanel.ts [hunk 1]
```diff
@@ -147,6 +147,11 @@ export async function trackMixpanelEvent(...[event, properties]: [...MixpanelEve
     mixpanel.track(event, { ...properties, namespace: namespaceFromEventName(event) });
   }
 }
+export async function timeMixpanelEvent(event: string) {
```
Comment (jaril) [inline src/ui/utils/mixpanel.ts:150]: Only the event is relevant here. @abstractalgo got some TS-fu that can get us the right type?

Comment (abstractalgo) [reply src/ui/utils/mixpanel.ts:150]: If I understand correctly, `event: MixpanelEvent[0]`?

Comment (jaril) [reply src/ui/utils/mixpanel.ts:150]: yep!

**Comment by jaril** [comment]
```
Also, we should mark every request for that recordingId+userId combo accepted since they could have requested multiple times (which I think is fine)
```

**Pull request merged**